web_frontend/osmaxx/excerptexport/views.py

```diff
@@ -11,6 +11,8 @@
 from django.utils.translation import ugettext_lazy as _
 from django.views.generic import FormView
 from django.conf import settings
+
+from config.settings.common import OSMAXX_FRONTEND_USER_GROUP
```

Comment:
don't import settings like this. Please use the overridable variant as stated in the [django documentation](https://docs.djangoproject.com/en/1.9/topics/settings/#using-settings-in-python-code).